File: h2o-docs/src/booklets/v2_2015/source/GLM_Vignette_code_examples/glm_fractionalbinomial_example.R
library(h2o)
h2o.init()
path = "https://s3.amazonaws.com/h2o-public-test-data/smalldata/glm_test/fraction_binommialOrig.csv"
train = h2o.importFile(path)
x <- c("log10conc")
y <- "y"

fractional_binomial <- h2o.glm (y = y, x = x, family = "fractionalbinomial", alpha = 0, lambda = 0, standardize = FALSE, compute_p_values = TRUE, training_frame = train)

File: h2o-docs/src/booklets/v2_2015/source/GLM_Vignette_code_examples/glm_fractionalbinomial_example.py
import h2o
from h2o.estimators.glm import H2OGeneralizedLinearEstimator
h2o.init()
train = h2o.import_file("https://s3.amazonaws.com/h2o-public-test-data/smalldata/glm_test/fraction_binommialOrig.csv")
x = ["log10conc"]
y = "y"

fractional_binomial = H2OGeneralizedLinearEstimator(family = "fractionalbinomial", alpha = [0], lambda_ = [0], standardize = False, compute_p_values = True)
fractional_binomial.train(x = x, y = y, training_frame = train)
